modules/dnn/src/libdnn/include/greentea_math_functions.hpp

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Thank you for the contribution, I believe, it will be very valuable!

Just a quick note while you are debugging the OpenCL kernels. We now have much stricter legal requirements, especially for the main repository. This heading comment has no license, has no copyright, we cannot include it into OpenCV in such a form. Besides, we are now regularly running tools to check the code cleanness. So, before the patch is integrated, we need to be sure that the proper header with the license and copyright is in place and the code in the file does not match with non-approved 3rd-party open-source software.